app/cypress/integration/cif/project-revision/index.spec.js

```diff
@@ -10,7 +8,7 @@ describe("the new project page", () => {
     cy.clock(new Date(2020, 5, 10), ["Date"]); // months are zero-indexed
   });
 
-  it("renders the project forms", () => {
+  it("renders the unfilled project forms", () => {
```

Comment:
At first I thought we could test some of the rendering with jest instead, but we're doing screenshots and accessibility here too